racket/btor.rkt

```diff
@@ -301,7 +300,7 @@
              (log-warning "Getting default value of 0 for bitvector, this may be a bad idea!")
              (bv 0 ,(string->number width-str))))]
         [`("sort" "array" ,index-sort-id-str ,element-sort-id-str)
-         (hash-set! sorts (string->number id-str) vector?)
+         (hash-set! sorts (string->number id-str) 'vector?)
```

Comment:
The only piece that is unclear to me is this `vector?`. If it's meant to be used in `define-symbolic`, `vector?` won't work because it's not a solvable type. But perhaps this should be addressed in a separate PR.